jbpm-services/jbpm-kie-services/src/main/resources/META-INF/Servicesorm.xml

```diff
@@ -714,12 +714,17 @@
 	      a.processId,
 	      a.processInstanceId,
 	      a.createdOn,
-	      a.dueDate
+	      a.dueDate,
+    	      a.workItemId,
+    	      log.slaDueDate,
+    	      log.slaCompliance
 	      )
         from 
-        	AuditTaskImpl a 
+        	AuditTaskImpl a,
+            	NodeInstanceLog log
```

Comment:
Not sure I like this join... would it be possible to store that info in AuditTask table to avoid join? Just asking ... it might not be that easy ..,

Reply:
@mswiderski for sure it would be better avoiding this join an it would be easier the console dataset query definition. Then, the proposal would be add this new two column to audiTask table and look for nodeInstanceDesc sla updates calls and propagate them to the auditTask too. Is that the idea?

Reply:
@mswiderski sorry I mean  nodeInstanceLog updates instead nodeInstanceDesc

Reply:
yes, that was a thought, do you think that would be doable?

Reply:
@mswiderski let me give it a try. Thanks!

Reply:
@nmirasch after looking it at more closely it will be rather difficult to make this without the join. Main reason is that SLA is tracked on process engine level (timers) and updates are only received within the process instance/node instance. So attempting to update the task tables will not really be an option without extra listeners etc. So please go ahead with the approach used in this PR, thanks!

Reply:
@mswiderski ook, Thanks!